Profiles - ZIB 2017/zib-LaboratoryTestResult-Observation.xml

```diff
@@ -501,19 +494,20 @@
     <element id="Observation.related">
       <path value="Observation.related" />
       <short value="RelatedResults" />
-      <definition value="Reference to related tests, e.g. paired tests or sequential tests like gram staining and microbiological cultures." />
-      <requirements value="Normally, an observation will have either a value or a set of related observations. A few observations (e.g. Apgar score) may have both a value and a set of related observations or sometimes QuestionnaireResponse from which the measure is derived." />
+      <definition value="Reference to related LaboratoryTestResult instances, or to the LaboratoryTest concept if this is a grouping Observation." />
+      <comment value="The Observation.related field is used for two different HCIM use cases:&#xD;&#xA;1. The LaboratoryTestResult can reference other, related LaboratoryTestResults (RelatedResult, NL-CM:13.1.33). In this case, `related.type` should be set to an appropriate value.&#xD;&#xA;2. The LaboratoryTestResult can consist of multiple LaboratoryTest concepts (NL-CM:13.1.3). In this case, there should be one parent instance of this profile that uses Observation.related to reference multiple child instances. `related.type` should in this case be set to &quot;has-member&quot;." />
       <alias value="GerelateerdeUitslag" />
+      <alias value="LaboratoriumTest" />
       <mapping>
         <identity value="hcim-laboratorytestresult-v4.1-2017EN" />
         <map value="NL-CM:13.1.33" />
         <comment value="RelatedResults" />
       </mapping>
-    </element>
-    <element id="Observation.related.type">
-      <path value="Observation.related.type" />
-      <min value="1" />
-      <fixedCode value="has-member" />
+      <mapping>
+        <identity value="hcim-laboratorytestresult-v4.1-2017EN" />
+        <map value="NL-CM:13.1.3" />
+        <comment value="LaboratoryTest" />
+      </mapping>

```

Comment:
Are we sure this mapping is correct here? The HCIM (RelatedResults) references to the entire HCIM. Otherwise, we should also change our wording at the StructreDefinition.comment as this also goes about the LaboratoryTestResult in stead of LaboratoryTest.

Reply:
Both mappings are present here. The agree with the comments, don't they?